skywalking/utils/ant_matcher.py

```diff
@@ -16,75 +16,24 @@
 #
 
 
-def fast_path_match(pattern: str, path: str):
```

Comment:
This is ported from a Java version, I've just run a simple benchmark test, regex is much more better then the original version, thanks